lua/autorun/server/iris_init.lua

```diff
@@ -0,0 +1,43 @@
+local baseURL = "http://iris.cfcservers.org/api"
+
+local function postJson( endpoint, data, callbackSuccess, callbackFailure )
+    HTTP{
+        failed = callbackFailure,
+        success = callbackSuccess,
+        method = "POST",
+        url = baseURL .. endpoint,
+        body = util.TableToJSON( data ),
+        type = "application/json"
+    }
+end
+
+local function logError( err )
+    print( "CFC Iris Interface HTTP error: " .. err)
+end
+
+local function sendGroupsToIris()
+    local userData = {}
+
+    for k, v in pairs( ULib.ucl.users ) do
+        userData[util.SteamIDTo64(k)] = v
+    end
+
+    postJson( "/ranks/bulk_update", {
+        users = userData,
+        realm = "cfc3",
+    }, nil, logError )
+end
+
+hook.Add( "InitPostEntity", "CFC_IrisInterface_BulkRankUpdate", sendGroupsToIris )
```

Comment:
need to test to see if ULib.ucl.users is loaded when this is called

Reply:
According to the wiki,
> `GM:InitPostEntity` - Good hook to create new entities. (You're still able to spawn them, but worldspawn won't be valid)

Makes me think it's fine